bk-config/bin/create_beekeeper_cert.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+mkdir -p /usr/lib/sage/bk-server
+
+
+if [ -e /usr/lib/sage/bk-server/id_rsa_sage_beekeeper-cert.pub ] ; then
+    echo "beekeeper cert already exists"
+    exit 0
+fi
+
```

Comment:
do you want

```
set -e
set -x
```

to be consistent with your other scripts?

Reply:
see comment to file create_beekeeper_cert.sh